src/workflows-app/components/InputsTable.test.js

```diff
@@ -106,8 +105,8 @@ describe('Input table rendering', () => {
     within(cells3[4]).getByDisplayValue('Hello World');
 
     // search for inputs belonging to target_workflow_1 task (removes foo_rating_workflow_var)
-    const searchInput = await screen.getByLabelText('Search inputs');
-    await userEvent.type(searchInput, 'target_wor');
+    const searchInput = screen.getByLabelText('Search inputs');
+    await user.type(searchInput, 'target_wor');
     await act(() => delay(300)); // debounced search
```

Comment:
It would be nice to mock the debounce so that tests don't have to spend this time waiting.